src/theme/themes/plh_kids_kw/_overrides.scss

```diff
@@ -1149,6 +1148,7 @@ body[data-theme="plh_kids_kw"] {
         .close-button {
           border: 0.8px solid var(--color-secondary-blue-80);
           color: var(--color-secondary-blue-50);
+          top: 0px !important;
```

Comment:
nit(non-blocking): might be good to add a comment to refer to this PR/issue for the important override

Reply:
It turns out the `!important` flag wasn't actually necessary, I was just anticipating it would be based on the overrides in the rest of the file